### PR TITLE
VACMS-4274: Make custom modules Drupal 9 compatible.

### PR DESCRIPTION
**Pull request by ndouglas**
## Description

See #4274. 

## Testing done
This issue has two main AC's.

The first of which is that we need to ensure that the line `core_version_requirement: ^8 || ^9` replaces `core: 8.x` in all custom modules.

This can be tested by running the following commands from the repo root:
1.  `grep -L "core_version_requirement" docroot/modules/custom/*/*.info.yml` -- search for files matching the latter pattern that _don't_ contain the new string -- this should return no results.
2. `grep "core:" docroot/modules/custom/*/*.info.yml` -- search for files that _do_ contain the old string -- this also should return no results.

The second is that we need to ensure the absence of any deprecated code.  This is not practical without [some static analysis system](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4231) or [module](https://www.drupal.org/project/upgrade_status).  Guess I'll add this to QA steps.

The only deprecated code in `va_gov_db` (some deprecated code exists in other modules and is handled by separate issues) is the following:
Screen Shot 2021-02-16 at 10 07 23 AM

in this code:

```php
$entity_manager = \Drupal::entityManager();
$previous_storage_definitions = $entity_manager->getLastInstalledFieldStorageDefinitions('node');
```

The `$entity_manager` variable is used only in the following call, and `getLastInstalledFieldStorageDefinitions()` should instead be called on the `entity.last_installed_schema.repository` service, so it's a trivial fix.

## Screenshots
Screen Shot 2021-02-16 at 9 26 26 AM

Screen Shot 2021-02-16 at 10 10 35 AM

## QA steps
1.  Go [here](https://pr4321-4tlphne45bmhzeaz1oeswpte7yfvyrd0.ci.cms.va.gov/admin/reports/upgrade-status).
2. Verify that the ACs are met:
    - [ ] All custom modules should have the above line added to their .info.yml files if it is not already present.
    - [ ] Custom module are not using deprecated methods (except for the `vagovadmin` theme, which is handled in a separate issue, none of our custom modules should show up under a "Fix With Rector" or "Fix Manually" header).

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review
- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?
- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner
- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by ndouglas** [comment]
Hey @acrollet mind giving this a look?  Just some YAML substitutions and two lines of code in an update hook.

**Comment by acrollet** [comment]
Not seeing any you missed:

Screen Shot 2021-02-16 at 11 54 52 AM

**Review by acrollet — APPROVED** [review]
approved pending tests

File: docroot/modules/custom/va_gov_db/va_gov_db.install [hunk 1]
```diff
@@ -367,8 +367,8 @@ function va_gov_db_update_8020() {
     }
   }
 
-  $entity_manager = \Drupal::entityManager();
-  $previous_storage_definitions = $entity_manager->getLastInstalledFieldStorageDefinitions('node');
+  $schema_repository = \Drupal::service('entity.last_installed_schema.repository');
+  $previous_storage_definitions = $schema_repository->getLastInstalledFieldStorageDefinitions('node');
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_db/va_gov_db.install:371]: This update hook ran its course back in Nov already.  Would have been no issue removing just gutting it and only leaving the shell so people know what the next N is.   Though this makes the example still available for for copying another time.

**Review by swirtSJW — APPROVED** [review]
Nice work @ndouglas

**Pull request merged**